webtau-browser/src/main/java/com/twosigma/webtau/browser/page/path/GenericPageElement.java

```diff
@@ -207,7 +207,7 @@ private String getUnderlyingValue() {
 
         for (int idx = 0; idx < elements.size(); idx++) {
             HtmlNode htmlNode = new HtmlNode(elementsMeta.get(idx));
-            PageElement pageElementByIdx = get(idx);
+            PageElement pageElementByIdx = get(idx + 1);
```

Comment:
Love off by one errors 😄 Is there a test that exercises this?

Reply:
there is end to end one that was broken.